### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by AdrienAVAL** [johngodley/redirection#646]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 5.3.10-1ubuntu3.14
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Cannot read property 'error_code' of undefined
Action: redirection/v1/redirect?orderby=id GET
Code: 403 Forbidden
Raw: [{"code":"json_cookie_invalid_nonce","message":"Le nonce du cookie n\u2019est pas valide"}]

Error: Cannot read property 'error_code' of undefined
Action: redirection/v1/group GET
Code: 403 Forbidden
Raw: [{"code":"json_cookie_invalid_nonce","message":"Le nonce du cookie n\u2019est pas valide"}]
```

**Comment by johngodley** [comment]
Please refresh the page and try again.

**Comment by AdrienAVAL** [comment]
I just refreshed the page but my problem is still there... 
Is it due to the extension update ?

**Comment by johngodley** [comment]
Can you try the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
